docsrc/index.rst

```diff
@@ -50,19 +35,3 @@ Need help? We have :ref:`active mailing lists <feedback-mailing-lists>`.
     operations
     developers
     support
-
---------
-
-===================
-What is Cyrus SASL?
```

Comment:
I think we should keep this section.  It's not so obvious from this repository, but this is a shared front page for the whole common website, not a Cyrus-IMAP specific page.